source/administration/compliance.rst

```diff
@@ -135,10 +135,14 @@ Compliance query results stored in ``posts.csv`` file
 | PostFilenames       | Comma separated list of filesnames attached to post           | ["/f../ho.png","/f../hi.png"] |
 +---------------------+---------------------------------------------------------------+-------------------------------+
 
-Global Relay Support 
-====================
+Global Relay Support (Beta)
```

Comment:
We didn't make any changes to the "Compliance" feature, only to "Compliance Exports".

Propose we add these changes to `compliance-export.rst` (see above note) + link to the new `compliance-export.rst` doc in here.